client/dns_record_list.go

```diff
@@ -0,0 +1,31 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+func (c *Client) ListDNSRecords(ctx context.Context, domain, teamID string) (r []DNSRecord, err error) {
+	url := fmt.Sprintf("%s/v4/domains/%s/records?limit=100", c.baseURL, domain)
```

Comment:
Could this be problematic if someone uses more than 100 domains?

Reply:
Good shout, it's currently only used by the sweeper (script to remove any orphaned test resources). So it shouldn't be an issue for anything.

Reply:
I'll add a comment explaining this though.